subsys/bluetooth/host/Kconfig

```diff
@@ -509,6 +509,14 @@ config BT_KEYS_SAVE_AGING_COUNTER_ON_PAIRING
 	  time a successful pairing occurs. This increases flash wear out but offers
 	  a more correct finding of the oldest unused pairing info.
 
+config BT_SMP_MIN_ENC_KEY_SIZE
+	int "Minimum encryption key size accepted in octets"
+	range 7 16
+	depends on !BT_SMP_SC_ONLY
```

Comment:
That's not quite what I had in mind when I talked about a conditional prompt. I meant the following, which doesn't require any additional pre-processor conditionals in the header file:
```
config BT_SMP_MIN_ENC_KEY_SIZE
        int
        prompt "Minimum encryption key size accepted in octets" if !BT_SMP_SC_ONLY
        range 7 16
        default 7
        default 16 if BT_SMP_SC_ONLY
```

Reply:
Ah, I see. Updated it. 

Btw, regarding the min key size; BT_SMP_SC_ONLY sets level 4 which doesn't use the min key size from smp.h, but simply hardcodes it to 128-bit. 

Nevertheless, adding this doesn't hurt :)